app/templates/main/logout.html

```diff
@@ -0,0 +1,10 @@
+{% macro logout_partial(sso_logout_url, get_help_url) %}
+<p class="flex-parent-element flexed-element-margins-collapse govuk-!-text-align-right">
+    <a href="{{ get_help_url }}"
+       class="govuk-link govuk-link--no-underline govuk-!-margin-right-5">
+        Get help</a>
+    <a href="{{ sso_logout_url }}" class="govuk-link govuk-link--no-underline">
+        Log out
+    </a>
+</p>
+{% endmacro %}
```

Comment:
Do we still want this partial if we're not using it yet? Maybe leave for a separate PR specific for Log Out/Get Help?